sopel/plugins/rules.py

```diff
@@ -1063,7 +1063,7 @@ def execute(self, bot, trigger):
         return exit_code
 
 
-class NamedRuleMixin:
+class AbstractNamedRule(Rule):
```

Comment:
An abstract class that inherits from a concrete one? Seems like one could instantiate an `AbstractNamedRule` without errors, which is probably not intended.

Reply:
Hm, yeah, right, I need to change that somehow.

Reply:
Since there is now an abstract method, you can't do that anymore. That's better. :)